src/V3FileLine.cpp

```diff
@@ -411,7 +411,7 @@ void FileLine::v3errorEnd(std::ostringstream& sstr, const string& extra)
     } else if (!V3Error::s().errorContexted()) {
         nsstr << warnContextPrimary();
     }
-    if (!m_waive) V3Waiver::addEntry(V3Error::s().errorCode(), filename(), sstr.str());
+    if (!warnIsOff(V3Error::s().errorCode()) && !m_waive) V3Waiver::addEntry(V3Error::s().errorCode(), filename(), sstr.str());
```

Comment:
The test works a little bit different to the other tests. 
The idea is that the first execution of verilator with the given .v file fails, since not all warinings are suppresed. The resulting waiver file, with the non supressed warnings is then used to generate a .vlt file in the obj_dir directory. Then the execution of the .v file is run again with the usage of the .vlt file. 
With the combination of the .vlt file and the usage of lint-off in the .v file the execution now runs through and does not fail. 

Is this idea behind the test also fine or would you like me to change it to the "classic" test with a static .vlt file? 
Also I wanted to ask if there is need for a test that tests the behavior when using --Wall? I can add this also to this pull request.

Reply:
Done